index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Tribute Page</title>
+</head>
+
+<body>
+    <div id="main">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <p>The man who saved a billion lives</p>
+        <div id="img-div"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" id="image">
```

Comment:
Add alt tag because ALT indicates the alternative text to display if the image itself cannot be displayed.

Reply:
- [ ] Please ask for any change in a more friendly and polite way by saying "Please add ..." or "Kindly add ...".
- [ ] Kindly you can tell the reviewee the reason why he needs to add `alt` attribute.